src/Symfony/Component/HttpFoundation/Tests/FileBagTest.php

```diff
@@ -60,6 +64,26 @@ public function testShouldSetEmptyUploadedFilesToNull()
         $this->assertNull($bag->get('file'));
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Passing a size as 4th argument to the constructor of "Symfony\Component\HttpFoundation\File\UploadedFile" is deprecated since Symfony 4.1 and will be unsupported in 5.0.
```

Comment:
I think we should not make these tests expect a deprecation, but update it to not trigger it (i.e. make the `UploadedFile` constructor treat the passed integer as the error code instead of the file size).

Reply:
see 7bb538bdca0b8c80dc781e6e2d4908179bd893d1